File: jdroid-gradle-googleplay-publisher-plugin/src/main/groovy/com/jdroid/android/googleplay/publisher/task/MetadataVerificationTask.java
package com.jdroid.android.googleplay.publisher.task;

import com.jdroid.android.googleplay.publisher.App;
import com.jdroid.android.googleplay.publisher.GooglePlayPublisher;

public class MetadataVerificationTask extends BaseTask {
	
	public MetadataVerificationTask() {
		setDescription("Verify that the content to upload to Google Play is valid.");
	}

	@Override
	protected void onExecute(App app) {
		GooglePlayPublisher.verifyMetadata(app);
	}

}

File: jdroid-gradle-googleplay-publisher-plugin/src/main/java/com/jdroid/android/googleplay/publisher/task/MetadataVerificationTask.kt
package com.jdroid.android.googleplay.publisher.task

import com.jdroid.android.googleplay.publisher.App
import com.jdroid.android.googleplay.publisher.GooglePlayPublisher

open class MetadataVerificationTask : BaseTask() {

    init {
        description = "Verify that the content to upload to Google Play is valid."
    }

    override fun onExecute(app: App) {
        GooglePlayPublisher.verifyMetadata(app)
    }
}
